packages/boot/test/bootstrapTests/test-addAssets.ts

```diff
@@ -0,0 +1,196 @@
+  const schedules = await EV(auctioneerKit.creatorFacet).getSchedule();
+  const { nextAuctionSchedule } = schedules;
+  t.truthy(nextAuctionSchedule);
+  const nextStartTime = nextAuctionSchedule.startTime;
+  const fiveMinutes = harden({
+    relValue: 5n * 60n,
+    timerBrand: nextStartTime.timerBrand,
+  });
+  const futureBusyTime = TimeMath.addAbsRel(nextStartTime, fiveMinutes);
+
+  await advanceTimeTo(futureBusyTime);
+
+  t.log('launching proposal');
+
+  const proposalJSON = JSON.stringify(proposal);
+  const proposalMod = proposalJSON
+    .replaceAll('STARS', 'PLANETS')
+    .replaceAll('ibc/987C17B1', 'ibc/987C17B2');
+  const proposalNew = JSON.parse(proposalMod);
+
+  const bridgeMessage = {
+    type: 'CORE_EVAL',
+    evals: proposalNew.evals,
+  };
+  t.log({ bridgeMessage });
+
+  const coreEvalBridgeHandler = await EV.vat('bootstrap').consumeItem(
+    'coreEvalBridgeHandler',
+  );
+  EV(coreEvalBridgeHandler).fromBridge(bridgeMessage);
+
+  const nextEndTime = nextAuctionSchedule.endTime;
+  const afterEndTime = TimeMath.addAbsRel(nextEndTime, fiveMinutes);
+  await advanceTimeTo(afterEndTime);
+
+  const schedulesAfter = await EV(auctioneerKit.creatorFacet).getSchedule();
+  // TimeMath.compareAbs() complains that the brands don't match
+  t.truthy(
+    schedules.nextAuctionSchedule.endTime.absValue <
+      schedulesAfter.nextAuctionSchedule.endTime.absValue,
+  );
+
+  t.like(readLatest(`${auctioneerPath}.book1`), { currentPriceLevel: null });
+});
+
+test('addAsset to auction starting soon', async t => {
+  const {
+    advanceTimeTo,
+    // @ts-expect-error override
+    proposal,
+    readLatest,
+  } = t.context;
+  const { EV } = t.context.runUtils;
+
+  const auctioneerKit = await EV.vat('bootstrap').consumeItem('auctioneerKit');
+  const schedules = await EV(auctioneerKit.creatorFacet).getSchedule();
+  const { nextAuctionSchedule } = schedules;
+  t.truthy(nextAuctionSchedule);
+  const nextStartTime = nextAuctionSchedule.startTime;
+  const fiveMinutes = harden({
+    relValue: 5n * 60n,
+    timerBrand: nextStartTime.timerBrand,
+  });
+  const tooCloseTime = TimeMath.subtractAbsRel(nextStartTime, fiveMinutes);
+
+  await advanceTimeTo(tooCloseTime);
+
+  const proposalJSON = JSON.stringify(proposal);
+  const proposalMod = proposalJSON
+    .replaceAll('STARS', 'MOONS')
+    .replaceAll('ibc/987C17B1', 'ibc/987C17B3');
+  const proposalNew = JSON.parse(proposalMod);
+
+  t.log('launching proposal');
+  const bridgeMessage = {
+    type: 'CORE_EVAL',
+    evals: proposalNew.evals,
+  };
+  t.log({ bridgeMessage });
+
+  const coreEvalBridgeHandler = await EV.vat('bootstrap').consumeItem(
+    'coreEvalBridgeHandler',
+  );
+  EV(coreEvalBridgeHandler).fromBridge(bridgeMessage);
+
+  const nextEndTime = nextAuctionSchedule.endTime;
+  const afterEndTime = TimeMath.addAbsRel(nextEndTime, fiveMinutes);
+  await advanceTimeTo(afterEndTime);
+
+  t.log('add-STARS proposal executed');
```

Comment:
s/STARS/MOONS, but you the logging will show it in the context of this test so simply,
```suggestion
  t.log('proposal executed');
```

Reply:
done